src/error.rs

```diff
@@ -29,6 +29,9 @@ pub enum HcHttpGatewayError {
     /// Error returned when a connection cannot be made to the upstream Holochain service
     #[error("The upstream Holochain service could not be reached")]
     UpstreamUnavailable,
+    /// Handles miscellaneous internal errors
+    #[error("Internal Error: {0}")]
+    InternalError(String),
```

Comment:
I'd prefer this split into specific errors. It looks like its being thrown for two distinct cases: invalid configuration, and failure to connect to admin websocket (which maybe should throw the existing `UpstreamUnavailable` variant).